proxies-es6/demo.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(receiver, name = 'world') {
+    return `Hello, ${name}!`;
+  }
+};
+
+var p = new Proxy(target, handler);
+ChromeSamples.log(p.folks);
+
+// Proxying a function object
+function sum(a, b) {
+  return a + b;
+}
+
+var handler2 = {
+  apply: function(target, thisArg, argumentsList) {
```

Comment:
You're using [method definition shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) in the previous handler and `function` here.

I'd recommend consistently using one or the other.